src/wallet/multisig-hd-wallet.js

```diff
@@ -1122,4 +1153,13 @@ export class MultisigHDWallet extends AbstractHDElectrumWallet {
   static isXpubForMultisig(xpub) {
     return ['xpub', 'Ypub', 'Zpub'].includes(xpub.substring(0, 4));
   }
+
+  /**
+   * @param mnemonic {string}  Mnemonic phrase (12 or 24 words)
+   * @returns {string} Hex fingerprint
+   */
+  static mnemonicToFingerprint(mnemonic) {
+    const seed = bip39.mnemonicToSeedSync(mnemonic);
+    return AbstractHDElectrumWallet.seedToFingerprint(seed);
+  }
```

Comment:
Are there unit tests to covers these new functions?

Reply:
@tanx The yarn lock info makes sense as all other git repos will need updates, hence removed the yarn instruction step from readmer.

Regarding mnemonic fingerprint: addCosigner  is the test case which is covering  mnemonicToFingerprint and addCosigner has a number of test cases.